version/1/4/th/code_of_conduct.md

```diff
@@ -0,0 +1,50 @@
+# จรรยาบรรณฉบับของกติกามารยาท 
+
+## คำมั่นสัญญาของเรา
+
+เพื่อส่งเสริมสภาพแวดล้อมที่เปิดกว้างและเป็นที่ยอมรับ 
+เราในฐานะผู้ร่วมให้การสนับสนุนและผู้ดูแล สัญญาว่าในโครงการและชุมชนของเราจะเป็นประสบการณ์ ที่ปราศจากอคติสำหรับทุกคน โดยไม่คำนึงถึง อายุ ขนาดของร่างกาย ความพิการ ชาติพันธุ์, เอกลักษณ์ทางเพศและการแสดงออก, ระดับประสบการณ์,สัญชาติ เชื้อชาติ และ ศาสนา
+
+## มาตรฐานของเรา
+
+ตัวอย่างของพฤติกรรมที่ก่อให้เกิดสภาพแวดล้อมที่ดี
+* ใช้ภาษาที่แสดงถึงการทำงานเป็นทีม ที่กระตุ้นให้เกิดสิ่งแวดล้อมที่ดี
+* เคารพในมุมมองและประสบการณ์ที่ต่างกัน
+* ยอมรับอย่างสุภาพในเชิงวิจารณ์ที่สร้างสรรค์
+* มุ่งเน้นที่สิ่งที่ดีที่สุดสำหรับชุมชน
+* แสดงความเห็นอกเห็นใจต่อสมาชิกชุมชนคนอื่น ๆ
+
+ตัวอย่างของพฤติกรรมที่ไม่สามารถยอมรับได้โดยผู้เข้าอบรม
+* การใช้ภาษาหรือภาพ ที่ส่อถึงการคุกคามทางเพศ 
+*การแสดงความคิดเห็น เพื่อการโจมตีส่วนบุคคลหรือทางการเมือง โดยไม่คำนึงถึงเหตุและผล
+* การล่วงละเมิดในที่สาธารณะหรือส่วนตัว
+* เผยแพร่ข้อมูลส่วนตัวของผู้อื่นเช่น ที่อยู่ อีเมล์ เฟสบุ๊ค โดยไม่ได้รับอนุญาต
+*พฤติกรรมอื่น ๆ ที่อาจถือว่าไม่เหมาะสมในการทำงานอย่างมืออาชีพ
+
+## ความรับผิดชอบของเรา
+
+ผู้ดูแลโครงการมีหน้าที่ในการชี้แจงมาตรฐานของพฤติกรรมที่เหมาะสม และมีบทลงโทษที่ยุติธรรมสำหรับการไม่ปฏิบัติตามกฎข้อตกลงดังกล่าวที่กำหนดไว้
+
+ผู้ดูแลโครงการมีสิทธิ์และหน้าที่รับผิดชอบในการลบ แก้ไขหรือ ปฏิเสธความคิดเห็น, commits, wiki edits โค้ด, , ปัญหาต่างๆและผลงานอื่น ๆที่ไม่สอดคล้องกับหลักจรรยาบรรณนี้หรือ ไม่อนุญาตให้มีส่วนร่วมอย่างชั่วคราวหรือถาวรสำหรับพฤติกรรมอื่น ๆ ที่เห็นว่าไม่เหมาะสม,เชิงข่มขู่, ล่วงละเมิดหรือเป็นอันตราย
```

Comment:
Remove comma, fix spacing and grammar
ในพจนานุกรม ฉบับราชบัณฑิตยสถาน ทั้งฉบับ พ.ศ. 2525 และฉบับ พ.ศ. 2542 เขียนไม้ยมกโดยเว้นวรรคทั้งหน้าและหลังไม้ยมก  เช่น  "ไม้ไผ่ลำยาว ๆ" และ "ติด ๆ กัน"